extension/src/plots/model/index.ts

```diff
@@ -266,6 +274,20 @@ export class PlotsModel extends ModelWithPersistence {
     return this.getSelectedComparisonPlots(paths, selectedRevisionIds)
   }
 
+  public getComparisonPlotClasses(paths: string[]) {
+    const selectedRevisionIds = this.getSelectedRevisionIds()
+    if (!definedAndNonEmpty(selectedRevisionIds)) {
+      return {}
+    }
+
+    return collectSelectedComparisonPlotClasses({
+      comparisonClassesSelected: this.getComparisonClassesSelected(),
```

Comment:
If this is empty, should this return `{}` as well? This would avoid having to loop through `selectedRevisionIds` for nothing.

Reply:
On further inspection, `comparisonClassesSelected` is only going to hold classes that the user has deselected/selected, it doesn't hold every visible class so we are unable to use it as a check for if there are selected classes or not.

Reply:
This variable probably needs to avoid confusion. Maybe something like `comparisonClassesSelectedState` 🤔

Reply:
No, `collectSelectedComparisonPlotClasses` could still return selected class boxes. Currently, we select all classes in a plot by default and `comparisonClassesSelected` only gets updated with a class (setting it as true or false) when a user clicks directly on a class button. Hope that makes sense :)